src/main/java/com/javadiscord/javabot/events/AutoMod.java

```diff
@@ -7,124 +7,140 @@
 import net.dv8tion.jda.api.EmbedBuilder;
 import net.dv8tion.jda.api.Permission;
 import net.dv8tion.jda.api.entities.Member;
-import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.MessageEmbed;
 import net.dv8tion.jda.api.entities.Role;
 import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;
 import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import org.jetbrains.annotations.NotNull;
 
 import java.awt.*;
 import java.util.Date;
-import java.util.List;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
-import java.util.stream.Collectors;
 
 public class AutoMod extends ListenerAdapter {
 
     private static final Pattern inviteURL = Pattern.compile("discord(?:(\\.(?:me|io|gg)|sites\\.com)/.{0,4}|app\\.com.{1,4}(?:invite|oauth2).{0,5}/)\\w+");
 
+    @Override
+    public void onGuildMessageReceived(@NotNull GuildMessageReceivedEvent event) {
+        try {
+            if (event.getMember().getUser().isBot() || event.getMember() == null)
+                return;
+        } catch (NullPointerException ignored) {
+            return;
+        }
```

Comment:
You don't need a try-catch here. Just check if `event.getMember()` is null before calling methods on it. For example, this is a more common "normal guild user" check:
```java
if (event.getMember() == null || event.getUser().isBot() || event.getUser().isSystem()) return;
```